CRM/Custom/Form/CustomDataByType.php

```diff
@@ -33,13 +33,13 @@ class CRM_Custom_Form_CustomDataByType extends CRM_Core_Form {
   public function preProcess() {
 
     $this->_type = $this->_cdType = CRM_Utils_Request::retrieve('type', 'String', CRM_Core_DAO::$_nullObject, TRUE);
-    $this->_subType = CRM_Utils_Request::retrieve('subType', 'String');
+    $subType = CRM_Utils_Request::retrieve('subType', 'String');
     $this->_subName = CRM_Utils_Request::retrieve('subName', 'String');
-    $this->_groupCount = CRM_Utils_Request::retrieve('cgcount', 'Positive');
+    $this->_groupCount = $groupID = CRM_Utils_Request::retrieve('cgcount', 'Positive');
```

Comment:
@eileenmcnaughton this doesn't look right it looks like this should be in L41 not here

Reply:
You are right!!

Reply:
@seamuslee001 fixed